elements/bulbs-dfp/bulbs-dfp.js

```diff
@@ -90,26 +90,19 @@ export default class BulbsDfp extends BulbsHTMLElement {
 
   handleInterval () {
     util.getAnalyticsManager().sendEvent({
-      eventCategory: 'bulbs-dfp-element Metrics',
+      eventCategory: 'bulbs-dfp-element Hot Metrics',
```

Comment:
Switching category to clearly separate from the earlier shadow metrics. These metrics affect real users.